### ISSUE TITLE
Subscriptions API: `HTTP DELETE /subscriptions/{id}`

### ISSUE THREAD

**Issue by birkjernstrom** [polarsource/polar#4595]
More context: https://github.com/polarsource/polar/issues/4594

**Comment by ofindley** [comment]
Hey, any info on a timeline or expected date of completion for this feature? Really need this feature. In the interim is there anyway for me to cancel the subscription of a user without them having a Polar account?

**Comment by birkjernstrom** [comment]
Started working on it now @ofindley and expect to launch it early next week at the latest. Ideally before, but it's a bit busy these days chatting with customers so don't want to promise it 🤓

**Comment by ofindley** [comment]
Thanks @birkjernstrom I really appreciate the response, looking forward to release my SaaS with payment handled by Polar.

**Comment by birkjernstrom** [comment]
Been busy with meetings & customer support so it's taken longer than anticipated, but close to done now and should go live today/tomorrow:  https://github.com/polarsource/polar/pull/4624

**Comment by amacfie** [comment]
Is this https://docs.polar.sh/api/v1/customer-portal/subscriptions/%7Bid%7D/delete or something else?

**Comment by birkjernstrom** [comment]
@amacfie No, it's directly a `DELETE` and `PATCH` of `/v1/subscriptions/{id}` allowing you to manage it with your PAT as the organization admin, i.e in your own backend/integration. Instead of having to use OAuth or similar and get the customer token to use via `/customer-portal/subscriptions/{id}`

Sorry for the delay on this one and not providing an update sooner. It was 99% done before the holidays, but decided to hold off on deploying it to avoid any remote chance of issues while everyone would be off. It's now been deployed @ofindley @amacfie

**Comment by ofindley** [comment]
Thanks @birkjernstrom
